lib/ruby-debug-ide/attach/util.rb

```diff
@@ -65,11 +66,20 @@ def get_child_pids(pid)
     pipe.readlines.each do |child|
       child_pid = child.strip.to_i
       q.push(child_pid)
-      pids << child_pid
+      pids.add(child_pid)
     end
   end
 
-  pids
+  pipe = IO.popen(%Q(lsof -c ruby | awk '{print $2 ":" $9}' | grep -E 'bin/ruby([[:digit:]]+\.?)*$'))
```

Comment:
a. lines 73-80 worth extracting into method
b. some debug logging on what are the child processes and which are filtered may be worth adding
c. please make another request with that commit (and sequent fix) since that one is merged already